### DISCUSSION TITLE
Unable to use Modin due to older version of numpy

### DISCUSSION THREAD

**Discussion by overseek944** [modin-project/modin#6090] [Q&A] [answered]
Hi,
I tried to use Modin but some of other the dependencies are causing conflict with numpy version that Modin is currently using. So I wanted to ask if there is any release of Modin available without numpy as its dependency or is there any way I can force Modin to use newer versions of Numpy. I am ok with any numpy version  >= 1.20.3.

Thanks.

**Comment by mvashishtha** [comment, marked as answer]
@overseek944 thank you for your comment. Our continuous integration (CI) tests run with the latest version of numpy, 1.24.3, and pass. I think numpy 1.24.3 should work. Could you please answer the following:

- which modin version are you using? (after importing modin, you can find this in `modin.__version__`)
- what are you using to install modin?
- which numpy version do you want to use?
- what is the error you get when you try to install or use modin?
